nix/home-config.nix

```diff
@@ -0,0 +1,31 @@
+# Instantiate homeConfigurations from standalone homes
+{
+  config,
+  inputs,
+  lib,
+  self,
+  withSystem,
+  ...
+}:
+let
+  homes = lib.flatten (map builtins.attrValues (builtins.attrValues config.den.homes));
+
+  homeConfiguration = home: {
+    inherit (home) name;
+    value = withSystem home.system (
+      { pkgs, ... }:
+      inputs.home-manager.lib.homeManagerConfiguration {
+        inherit pkgs;
+        modules = [ self.modules.${home.class}.${home.aspect} ];
+      }
+    );
+  };
+
+  homeConfigurations =
+    class: builtins.listToAttrs (map homeConfiguration (builtins.filter (x: x.class == class) homes));
+
+  flake.homeConfigurations = homeConfigurations "homeManager";
```

Comment:
this is correct, however it looks like our current flake checks on the default template are not verifying the homeConfiguration. I guess because the default flake has no dependency on homeManager itself, can you please add another module inside `_example` that adds `flake-file.inputs.home-manager` dependency ? so we can properly check the example configuration ?